src/label_projection/control_methods/baseline_random_celltype/config.vsh.yaml

```diff
@@ -0,0 +1,42 @@
+functionality:
+  name: "baseline_random_celltype"
+  namespace: "label_projection/control_methods"
+  version: "dev"
+  description: "Random Labels dummy"
+  authors:
```

Comment:
Please add metadata regarding:
* whether method is a baseline, positive control, negative control or a regular method
* nice to read method label

See [neurips2021_multimodal_viash](https://github.com/openproblems-bio/neurips2021_multimodal_viash/blob/main/src/match_modality/methods/dummy_random/config.vsh.yaml#L6-L8) for an example